Pod/Classes/PINURLSessionManager.m

```diff
@@ -72,6 +72,11 @@ - (void)unlock
 
 #pragma mark NSURLSessionDataDelegate
 
+- (void)URLSession:(NSURLSession *)session dataTask:(NSURLSessionDataTask *)dataTask didReceiveResponse:(NSURLResponse *)response completionHandler:(void (^)(NSURLSessionResponseDisposition))completionHandler
```

Comment:
Can you add a comment explaining why this is implemented? That way someone won't remove this in the future?